src/App.js

```diff
@@ -1,25 +1,45 @@
 import React, { useState, useEffect } from 'react';
-import { Switch, Route, Redirect, Link } from 'react-router-dom';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import './App.css';
+import 'animate.css';
+
 import requestSentiment from '../src/util/apiCalls';
 import faceSwitch from '../src/util/faceSwitch';
 
 import Form from './components/Form/Form';
+import Drafts from './components/Drafts/Drafts';
+import Result from './components/Result/Result';
+import Title from './components/Title/Title';
+import Emoji from './components/Emoji/Emoji';
 
 export default function App() {
   const [drafts, setDrafts] = useState([]);
```

Comment:
Great use of Hooks on this file. They certainly streamline things. Get familiar with useContext() as well for next mod. For more complicated multi-page apps it can be a very useful alternative to Redux for managing state, depending on the use case.